packages/grafana-ui/src/components/PanelChrome/PanelChrome.story.tsx

```diff
@@ -105,4 +109,48 @@ export const Examples = () => {
   );
 };
 
+export const Basic: ComponentStory<typeof PanelChrome> = (args: PanelChromeProps) => {
+  const theme = useTheme2();
+
+  const contentStyle: CSSProperties = {
+    background: theme.colors.background.secondary,
```

Comment:
> in light mode the child's text is not readable

This is something specific to these stories, because toggling the Storybook theme doesn't actually change the theme that is passed to the component. The solution is to add `color: theme.colors.text.primary` here, so that both background and color use the theme that is passed in.

Reply:
Cool, missed that!